src/content/docs/infrastructure/host-integrations/understand-use-data/host-integration-data-collection-reporting.mdx

```diff
@@ -25,8 +27,24 @@ This is how an infrastructure on-host integration sends data to New Relic:
 5. At the scheduled interval (the default is 15 seconds), the agent harvests the data from the integration and prepares it for transmission. Every 60 seconds, it sends that data to New Relic, along with any other infrastructure data.
 6. After a successful collection pass, the integration executable exits.
 
+<img
+  title="Agents and New Relic"
+  alt="Agents and New Relic"
+  src={infrastructureAgentsandNr}
+/>
+
+## Integration architecture [#example-diagram]
```

Comment:
@zuluecho9 where did this section come from? It seems incomplete. Is there a diagram for this?